src/Tribe/Views/V2/Views/Traits/HTML_Cache.php

```diff
@@ -196,4 +200,152 @@ protected function should_enable_html_cache( $context ) {
 		// In all other cases, let's cache it!
 		return true;
 	}
+
+	/**
+	 * Get the list of fields/input we will do replacement for HTML Cache.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @return array   List of fields/input that we are going to replace.
+	 */
+	protected function get_view_nonce_fields() {
+		$nonces = [
+			'wp_rest' => 'tribe-events-views[_wpnonce]',
+		];
+
+		/**
+		 * Filter to control nonce fields replacement for HTML Cache.
+		 *
+		 * @since 5.0.0
+		 *
+		 * @param array   $nonces   List of action and field name where the nonce is stored.
+		 * @param Context $context  Context from the current view.
+		 * @param View    $view     Which view instance we are currently rendering.
+		 */
+		return apply_filters( 'tribe_events_views_v2_get_view_nonce_fields', $nonces, $this->get_context(), $this );
+	}
+
+	/**
+	 * Get the list of attributes we will do replacement for HTML Cache.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @return array   List of attributes that we are going to replace.
+	 */
+	protected function get_view_nonce_attributes() {
+		$nonces = [
+			'wp_rest' => 'data-view-rest-nonce',
+		];
+
+		/**
+		 * Filter to control nonce attributes replacement for HTML Cache.
+		 *
+		 * @since 5.0.0
+		 *
+		 * @param array   $nonces   List of action and field name where the nonce is stored.
+		 * @param Context $context  Context from the current view.
+		 * @param View    $view     Which view instance we are currently rendering.
+		 */
+		return apply_filters( 'tribe_events_views_v2_get_view_nonce_attributes', $nonces, $this->get_context(), $this );
+	}
+
+	/**
+	 * Get the list of JSON properties we will do replacement for HTML Cache.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @return array   List of json properties that we are going to replace.
+	 */
+	protected function get_view_nonce_json_properties() {
+		$nonces = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: bed675c8f8609534c257d73cb94a70d0</sub>